Userland/Libraries/LibGUI/TextEditor.cpp

```diff
@@ -893,6 +898,11 @@ void TextEditor::keydown_event(KeyEvent& event)
         }
     }
 
+    if (is_multi_line() && !event.shift() && !event.alt() && event.ctrl() && event.key() == KeyCode::Key_F) {
+        m_search_banner->show();
+        return;
+    }
```

Comment:
My instinct says this should be an Action, but I'm not sure if that would actually make sense. That can be worried about later though.